src/components/FavoriteActivities/index.tsx

```diff
@@ -69,7 +69,7 @@ const FavoriteActivities = () => {
           {cards.map((card, index) => (
             <Card
               key={card.id}
-              icon={card.icon}
+              icon={card.image}
```

Comment:
It's all good with the change, however the component name should follow the British grammar. Can you update it to `FavouriteActivities`?